lib/simple_rich_text.dart

```diff
@@ -161,109 +216,132 @@ class SimpleRichText extends StatelessWidget {
 //            },
 //          )));
 
-            assert(context != null, 'must pass context if using route links');
+              assert(context != null, 'must pass context if using route links');
 
-            onTapNew(String caption, Map m) {
-              assert(m != null);
-              if (map.containsKey('push')) {
-                String v = map['push'];
-                return () {
-                  if (log ?? false) print("TAP: PUSH: $caption => /$v");
-                  assert(v != null);
+              onTapNew(String caption, Map m) {
+                assert(m != null);
+                if (map.containsKey('push')) {
+                  String v = map['push'];
+                  return () {
+                    if (log ?? false) print("TAP: PUSH: $caption => /$v");
+                    assert(v != null);
 //                  Navigator.pushNamed(context, '/$v');
-                  Nav.push('/$v');
-                };
-              } else if (map.containsKey('repl')) {
-                String v = map['repl'];
-                return () {
-                  if (log ?? false) print("TAP: POP&PUSH: $caption => /$v");
-                  assert(v != null);
+                    Nav.push('/$v');
+                  };
+                } else if (map.containsKey('repl')) {
+                  String v = map['repl'];
+                  return () {
+                    if (log ?? false) print("TAP: POP&PUSH: $caption => /$v");
+                    assert(v != null);
 //                  Navigator.popAndPushNamed(context, '/$v');
-                  Nav.repl('/$v');
-                };
-              } else {
-                return () {
-                  if (log ?? false) print("TAP: $caption => pop");
+                    Nav.repl('/$v');
+                  };
+                } else if (map.containsKey('http')) {
+                  String v = map['http'];
+                  return () async {
+                    print("TAP: HTTP: $caption => /$v");
+                    assert(v != null);
+                    try {
+                      await launch('http://$v');
+                    } catch (e) {
+                      print('Could not launch http://$v: $e');
+                      try {
+                        await launch('https://$v');
+                      } catch (e) {
+                        print('Could not launch https://$v: $e');
+                      }
+                    }
+                  }; // TODO add possibility of tel, mailto, sms, whats,...?
```

Comment:
`url_launcher` can handle not only `http`

I propose to use `link` instead of `http` and handle any links